ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts

```diff
@@ -64,25 +74,24 @@ export function getLineSeries(
     stack: visual.stack === 'All' ? visual.stack : undefined,
     sampling: 'lttb',
     progressiveThreshold: OPTIMIZED_MODE_SERIES_LIMIT, // https://echarts.apache.org/en/option.html#series-lines.progressiveThreshold
-    showSymbol: visual.show_points === 'Always' ? true : false,
+    showSymbol: showPoints,
+    showAllSymbol: true,
     symbolSize: pointRadius,
     lineStyle: {
       width: lineWidth,
+      opacity: 0.85,
     },
     areaStyle: {
       opacity: visual.area_opacity ?? DEFAULT_AREA_OPACITY,
     },
+    // https://echarts.apache.org/en/option.html#series-line.emphasis
     emphasis: {
-      focus: 'series',
-      blurScope: 'coordinateSystem',
+      focus: 'none', // 'none', 'self', 'series'
```

Comment:
Is this not needed anymore because you're handling emphasis above?

Reply:
Optional nit: It might be more helpful to have a comment clarifying why you set the value to "none" than the other possible values.

Reply:
I ended up removing these since the defaults will work fine for now. 

I do want to try bringing back `blur` again later so we have more control over the opacity fading out for non-focused series (it's tricky to get this right with the custom 'downplay' actions and the current state is already a big improvement over what's on our main branch)